src/as/core/Geometry.spec.ts

```diff
@@ -136,17 +136,17 @@ describe('Geometry', () => {
 		var v0 = geometry.vertices[0],
 			v1 = geometry.vertices[1],
 			v2 = geometry.vertices[2]
+
 		// assert(v0.x === 0.5 && v0.y === 0 && v0.z < Number.EPSILON, 'first vertex was rotated')
-		//expect<bool>(v0.x === 0.5 && v0.y === 0 && v0.z < Number.EPSILON).toBe(true)
+		//expect<bool>(v0.x === 0.5 && v0.y === 0 && v0.z < f32.EPSILON).toBe(true)
 		expect<bool>(v0.x === 0.5 && v0.y === 0 && v0.z < Mathf.pow(2, -52)).toBe(true)
 		// assert(v1.x === -0.5 && v1.y === 0 && v1.z < Number.EPSILON, 'second vertex was rotated')
-		//expect<bool>(v1.x === -0.5 && v1.y === 0 && v1.z < Number.EPSILON).toBe(true)
+		expect<bool>(v1.x === -0.5 && v1.y === 0 && v1.z < f32.EPSILON).toBe(true)
 		// expect<bool>(v1.x === -0.5 && v1.y === 0 && v1.z < Mathf.pow(2, -52)).toBe(true, 'v1 values passed')
-		expect(v1.x === -0.5).toBe(true, 'v1 x')
-		expect(v1.y === 0).toBe(true, 'v1 y')
-		//expect(v1.z < f32(Mathf.pow(2, -52))).toBe(true, 'v1 z')
+		// expect(v1.x === -0.5).toBe(true, 'v1 x')
+		// expect(v1.y === 0).toBe(true, 'v1 y')
+		// expect(v1.z < f32(Mathf.pow(2, -52))).toBe(true, 'v1 z')
```

Comment:
Just curious, did you leave some of them commented, but planning to work on once we merge this one?

Reply:
Looks like the `|0` just floors the value. Is that was the `u32()` cast does too?

Reply:
Why isn't this expect(v1.x).toBe(-0.5)?

Reply:
It will be!